src/scripts/browserDeviceProfile.js

```diff
@@ -210,12 +210,18 @@ function supportsDolbyVision(options) {
     );
 }
 
-function canPlayDolbyVisionHevc(videoTestElement) {
-    // Profiles 5/7/8 4k@60fps
-    return !!videoTestElement.canPlayType
-        && (videoTestElement.canPlayType('video/mp4; codecs="dvh1.05.09"').replace(/no/, '')
-        && videoTestElement.canPlayType('video/mp4; codecs="dvh1.07.09"').replace(/no/, '')
-        && videoTestElement.canPlayType('video/mp4; codecs="dvh1.08.09"').replace(/no/, ''));
+function supportedDolbyVisionProfilesHevc(videoTestElement) {
+    const supportedProfiles = [];
+    // Profiles 5/8 4k@60fps
+    if (videoTestElement.canPlayType) {
+        if (videoTestElement
+            .canPlayType('video/mp4; codecs="dvh1.05.09"')
+            .replace(/no/, '')) supportedProfiles.push(5);
+        if (videoTestElement
+            .canPlayType('video/mp4; codecs="dvh1.08.09"')
+            .replace(/no/, '')) supportedProfiles.push(8);
```

Comment:
The bodies of these long multi-line if statements should be on a separate line (and wrapped in braces).